restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentDB.java

```diff
@@ -141,16 +145,29 @@ public boolean treatBooleansAsNumeric() {
     return allColumns;
   }
 
-  public QueryBuilder builder() {
+  public QueryBuilder queryBuilder() {
     return dataStore.queryBuilder();
   }
 
   public Schema schema() {
     return dataStore.schema();
   }
 
+  public Table getTable(String keyspaceName, String table) {
+    Keyspace keyspace = getKeyspace(keyspaceName);
```

Comment:
why not `dataStore.getTable()`?

Reply:
I can do that too; just copied code. Will change, no real reason (and some consider it bad practice for public API methods to call other public API methods).

Reply:
Oh. Actually, no; `DataStore` has just accessor for `schema`; and `schema` for `Keyspace`.
So there's no convenience method. I could add method there if that makes sense but I think for this PR I'll leave it as-is.